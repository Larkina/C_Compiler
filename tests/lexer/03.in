// YTESTLKST


   /* aasd */ ababaca /* end comments */