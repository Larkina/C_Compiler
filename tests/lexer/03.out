value=ababaca, text=ababaca, type=VAR
